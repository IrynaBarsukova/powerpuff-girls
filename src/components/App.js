import React from 'react';
import './App.scss';

function App() {
    return (
        <div className="App">
            <header className="App-header">Hello world</header>
        </div>
    );
}

export default App;
