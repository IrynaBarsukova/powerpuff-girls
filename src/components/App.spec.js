import React from 'react';
import { Provider } from 'react-redux';

import MockStore from '../mocks/store.mock';
import renderWithRouter from '../mocks/router.mock';
import App from './App';

describe('<App />', () => {
    let store;

    beforeEach(() => {
        store = new MockStore({});
    });

    it('should render App', () => {
        const { container } = renderWithRouter(
            <Provider store={store}>
                <App />
            </Provider>,
        );

        expect(container).toMatchSnapshot();
    });
});
