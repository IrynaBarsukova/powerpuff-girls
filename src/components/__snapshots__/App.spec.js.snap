// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> should render App 1`] = `
<div>
  <header
    class="header"
  >
    <div
      class="container"
    >
      <a
        class="header__link"
        href="/"
      >
        <h1
          class="header__title"
        >
          The Powerpuff Girls
        </h1>
        <img
          alt="Logo"
          class="header__logo"
          src="[object Object]"
        />
      </a>
    </div>
  </header>
  <div
    class="home"
  >
    <div
      class="container"
    >
      <a
        class="home__link"
        href="/shows/6771"
      >
        Watch show
      </a>
    </div>
  </div>
</div>
`;
